core/storage/topic/gae_models.py

```diff
@@ -107,6 +107,9 @@ class TopicModel(base_models.VersionedModel):
     # The content of the meta tag in the Topic viewer page.
     meta_tag_content = datastore_services.StringProperty(
         indexed=True, default='')
+    # The page title used in the Topic viewer page.
+    page_title = datastore_services.StringProperty(
```

Comment:
My initial impression is that this feels a bit too much like 'UI' to be stored in the datastore. See my reply to the earlier comment thread on maybe using a name that better ties to the domain layer rather than something that's UI-specific.

Reply:
Done, changed it to "page_title_for_web".